packages/e2e/features/ui/hooks/actions-navigation.feature

```diff
@@ -19,4 +19,4 @@ Feature: Navigate action hooks
   @react
   Scenario: Navigation location change action works correctly
     When I click the "Go to amazon.com" button
-    Then My url contains "ui/hooks/actions?pageChange"
+    Then My url contains "ui/hooks/actions/?pageChange"
```

Comment:
This is to account for `trailingSlash: true` that we added in https://github.com/aws-amplify/amplify-ui/pull/2704/files#r989583305.